internal/actor/actoruid/uid.go

```diff
@@ -8,6 +8,9 @@ import (
 
 const prefix = "urn:opg:poas:makeregister:users:"
 
+// Service is used when the application does an action.
+var Service = UID{value: [16]byte{0, 0, 0, 0, 0, 0, 64}}
```

Comment:
I didn't want to use the null UUID here, so settled for setting the version to 4. We can always change this later if/when we decide how to represent it. But it shouldn't really matter too much just for getting an LPA as that isn't recorded anywhere.